core/include/traccc/seeding/track_params_estimation.hpp

```diff
@@ -1,14 +1,13 @@
 /** TRACCC library, part of the ACTS project (R&D line)
  *
- * (c) 2021-2022 CERN for the benefit of the ACTS project
+ * (c) 2021-2024 CERN for the benefit of the ACTS project
  *
  * Mozilla Public License Version 2.0
  */
 
 #pragma once
 
 // Library include(s).
-#include "traccc/edm/cell.hpp"
```

Comment:
This file does not need the new, updated cell EDM?

Reply:
Yepp. A couple of files didn't need this include at all.